cmd/prometheus/main.go

```diff
@@ -265,6 +288,9 @@ func main() {
 	a.Flag("query.max-samples", "Maximum number of samples a single query can load into memory. Note that queries will fail if they try to load more samples than this into memory, so this also limits the number of samples a query can return.").
 		Default("50000000").IntVar(&cfg.queryMaxSamples)
 
+	a.Flag("enable-feature", "Comma separated feature names to enable. Valid options: (1) 'promql-at-modifier' to enable the @ modifier.").
```

Comment:
What's the (1) about?

Reply:
Because we will have (2) shortly after I guess, the negative offset. Hence doing the ground work here :)

Reply:
Number just make it easier to parse the options since we also have the description for options, a comma is immediately less visible when you have multiple options. But I guess the quotes around the option will make it easily noticeable. I will remove (1) for now.

Reply:
I can assume this list to grow quite a bit in the future. I also think many of these feature flags need explanations about implications. How about linking to a separate file explaining the current options? Or put the detailed explanations behind a `--help` flag?

Reply:
I think detailed stuff an explanation would best go in the docs somewhere.

Reply:
And I think adding a separate section in docs mentioning all the disabled features is also useful for more visibility into them so that more people try it out and we could promote the feature to be default sooner.